### DISCUSSION TITLE
Software change request - creating virtual individuals and populations with obesity

### DISCUSSION THREAD

**Discussion by jgg1822** [Open-Systems-Pharmacology/Forum#757] [Questions & Problems] [answered]
I’m posting to request a change to the software to better allow for creating virtual pediatric individuals and populations with obesity in PK-Sim.

Because children with obesity are defined by BMI percentile rather than raw BMI, it is difficult to create virtual children with obesity in the current software. Currently, the probability of a virtual subject being obese is irrespective of age, which means that there is an uneven age and obesity distribution.

I propose having some type of ‘Obese’ selection in the Individual and Population building blocks with filters in the back-end, as well as a user-input field for BMI percentile in the Population building block so that the user can create populations with varying body size categories (i.e. overweight, obese, morbidly obese, etc).

Note that these changes are only relevant for virtual subjects 2 years or older, below which BMI and obesity status cannot be determined.

**Comment by Aedginto** [comment, marked as answer]
Hi JGG,
As I know you are working on the differential and age-dependence of anatomy/physiology in obese pediatric populations, we will need to figure out how to incorporate this information when we create obese individuals. 
To change the anatomy/physiology as a function of obesity would require, in my opinion, two additions:
1) incorporating the change [e.g. 15% increase in liver weight when BMI is > 95th percentile of the population (definition of obesity) from which the individual comes] in the algorithm to 'Create Individual'. This allows for an obese individual to be created with the associated anatomical/physiological changes.
2) ability to filter by BMI percentile, not absolute BMI, when creating a population. As you suggest, filtering by absolute BMI will favour those in the pediatric population with naturally lower BMI because the probability of creating them is higher. This results in a population with many young kids whose BMI is naturally lower who may not even be obese. As a result, to ensure that the probability of getting an even age distribution of actually obese individuals in the pediatric population, filtering by BMI percentile will be necessary. 

Also, by doing 1) and 2), an obese population can be created for any race where there are populations in PK-Sim. It avoids having just one 'Obese Population' that is fixed and irrespective of race. 

There are probably a number of ways to do this. Let's brainstorm to figure out what's best!
Andrea
